Endzone.Umbraco.Extensions/PublishedContentExtensions/MultiUrlPicker.cs

```diff
@@ -40,6 +40,41 @@ public static Link GetLink(this IPublishedContent content, string property, bool
         }
 
         /// <summary>
+        /// Gets the links as a collection of IPublishedContent. Check for null.
+        /// </summary>
+        /// <param name="content"></param>
+        /// <param name="property"></param>
+        /// <param name="recursive"></param>
+        /// <returns></returns>
+        public static IEnumerable<IPublishedContent> GetLinksContent(this IPublishedContent content, string property, bool recursive = false)
+        {
+            var umbracoHelper = new UmbracoHelper(UmbracoContext.Current);
+            var links = content.GetLinks(property, recursive);
+            var linksContent = new List<IPublishedContent>();
+            foreach (var link in links)
+            {
+                if (link != null)
+                {
+                    linksContent.Add(umbracoHelper.TypedContent(link.Id));
```

Comment:
Is this potentially unsafe when the link is external? I think an additional check needs to be added to the if statement above.